modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/BaseDDMTestCase.java

```diff
@@ -112,18 +117,10 @@
  * @author Pablo Carvalho
  * @author Miguel Angelo Caldas Gallindo
  */
-@PrepareForTest(
-	{
-		DDMStructureLocalServiceUtil.class, DDMTemplateLocalServiceUtil.class,
-		LocaleUtil.class, PortalClassLoaderUtil.class, PropsValues.class,
-		ResourceBundleUtil.class
-	}
-)
+@PrepareForTest({LocaleUtil.class, PropsValues.class})
 @RunWith(PowerMockRunner.class)
```

Comment:
@JanisZhang There is still usage of powermock in this class.

Tina.

Reply:
@tinatian yes, I think it can not be replaced, I will rework this class after upgrading to Mockito.